lib/block_view.go

```diff
@@ -3581,7 +3598,7 @@ func (bav *UtxoView) _connectTransaction(
 			err, "_connectTransaction: Problem serializing transaction: ")
 	}
 	txnSizeBytes := uint64(len(txnBytes))
-	if txnSizeBytes > bav.Params.MaxBlockSizeBytes/2 {
+	if txnSizeBytes > bav.GetMaxBlockSizeBytes(uint64(blockHeight))/2 {
```

Comment:
See comment below
```suggestion
	if txnSizeBytes > bav.GetCurrentGlobalParamsEntry().MaxBlockSizeBytesPoS/2 {
```

Reply:
This will break PoW. we need to support having two different values.